README.md

```diff
@@ -295,9 +285,44 @@ Similarly, you can also write sequence schemas that expect particular values in
 (s/validate StringListOrKeywordMap [:foo])
 ;; RuntimeException:  Value does not match schema: [(not (instance? java.lang.String :foo))]
 
+;; if (shorthand for conditional)
+(def StringListOrKeywordMap (s/if map? {s/Keyword s/Keyword} [String]))
+
+;; cond-pre (experimental), also shorthand for conditional, allows you to skip the
+;; predicate when the options are superficially different
+(def StringListOrKeywordMap (s/cond-pre {s/Keyword s/Keyword} [String]))
+;; but don't do this (use `if` or `abstract-map-schema` here instead):
+(def BadSchema (s/cond-pre {:a s/Keyword} {:b s/Keyword}))
+
+;; conditional can also be used to apply extra validation to a single type
+(def OddLong (s/conditional odd? long))
```

Comment:
probably not the best example, as validating non-long value will throw from `odd?`